builder/base/handlebars/kss_builder_base_handlebars.js

```diff
@@ -427,7 +434,7 @@ class KssBuilderBaseHandlebars extends KssBuilderBase {
       if (section.markup) {
         // Load the information about this section's markup partial.
         let partialInfo = this.partials[section.reference];
-        let template = this.Handlebars.compile('{{> "' + partialInfo.name + '"}}');
+        let template = this.Handlebars.compile('{{> "' + partialInfo.name + '"}}', {noEscape: true});
```

Comment:
I believe John was refering to this one. Rather then `{{`, use the three braces.